rasa/core/policies/unexpected_intent_policy.py

```diff
@@ -846,26 +866,31 @@ def _load_model_utilities(cls, model_path: Path) -> Dict[Text, Any]:
 
     @classmethod
     def _update_loaded_params(cls, meta: Dict[Text, Any]) -> Dict[Text, Any]:
-        meta = train_utils.override_defaults(cls.defaults, meta)
+        meta = train_utils.override_defaults(cls.get_default_config(), meta)
```

Comment:
we don't need this any longer as the `GraphNode` (see [here](https://github.com/RasaHQ/rasa/blob/5bb331ff7ba75ad564478cbc71fd1a46bce5ee02/rasa/engine/graph.py#L316)) is taking care of it
```suggestion
```